dev/docker/scripts/ursula.ps1

```diff
@@ -0,0 +1 @@
+docker run -it dev:nucypher nucypher ursula run --dev --federated-only
```

Comment:
What is the best way to view file logging inside this process?

Reply:
this basically will open an ursula "console" just as if you ran that `nucypher nucypher ursula run --dev --federated-only` command in a regular shell.  You would know the answer to that better than I would and I'd love your help setting all this stuff up to work optimally.

Reply:
OK - got it - I think I was confused as to where this is running.